tests/phpMyFAQ/Controller/Administration/Api/FaqControllerTest.php

```
@@ -2229,24 +2553,119 @@ public function testCreateSucceedsForInactiveFaqWithoutPublishPermission(): void
+            ->method('create')
+            ->willReturnCallback(static fn(\phpMyFAQ\Entity\FaqEntity $faqEntity): \phpMyFAQ\Entity\FaqEntity => $faqEntity->setId(1));
+
+        $controller = $this->createControllerWithFaq($faq);
+        $controller->setContainer($this->createAuthenticatedContainer($session));
+
+        foreach (['draft', 'review'] as $status) {
+            $request = $this->createRequestForNewFaq($csrfToken, status: $status);
+
+            $response = $controller->create($request);
+
+            self::assertSame(Response::HTTP_OK, $response->getStatusCode(), 'status: ' . $status);
```

PHPStan (2.2.7)

[error] Call to an undefined static method phpMyFAQ\Controller\Administration\Api\FaqControllerTest::assertSame().


(staticMethod.notFound)